### DISCUSSION TITLE
Glances widget not loading when using Dashy with Traefik or Nginx

### DISCUSSION THREAD

**Discussion by ViciousWall** [Lissy93/dashy#687] [Help and Support] [answered]
Just want to start with saying I really love this dashboard and thank you Lissy93 for all the work you have put in to make this app great.

I recently tried adding Glances widget with my dashboard and for some reason I keep getting an error loading the widget when I run the service using Traefik or Nginx but when I use my internal IP it loads without any issues. I am not sure if I am doing something wrong here but any help on this would be appreciated.

Let me know if you need any screenshots or any other information on this.

Thanks again :)

**Comment by lissy93** [comment, marked as answer]
Have replied here: #686 :)
